docs/guide/configure.md

```diff
@@ -206,6 +206,10 @@ The map section lets you customize how maps appear and behave across GeoNetwork-
 
   Will limit the possibility to zoom in past a certain zoom level.
 
+- `do_not_tile_wms` (optional)
+
+  Will not tile WMS. Defaults to `false` (WMS are tiled).
```

Comment:
```suggestion
  Will not use tiling when requesting WMS services. Defaults to `false` (WMS are tiled). Not using tiles for WMS might incur performance loss since the client will not benefit from an eventual tile cache anymore. On the other hand, visual quality might improve in case a map tile server does not handle neighbouring tiles correctly, e.g. symbols or text being cropped at tile boundaries.
```